dev-client/src/config/index.ts

```diff
@@ -32,9 +33,21 @@ export const getConfig = apiConfig;
 type AppConfig = {
   packageName: string;
   googleClientId: string;
+  googleRedirectURI: string;
 };
 
 export const APP_CONFIG: AppConfig = {
   packageName: 'org.terraso.landpks',
-  googleClientId: Config.GOOGLE_OAUTH_APP_CLIENT_ID ?? '',
+  googleClientId:
+    Platform.OS === 'ios'
+      ? Config.GOOGLE_OAUTH_IOS_CLIENT_ID ?? ''
+      : Platform.OS === 'android'
+      ? Config.GOOGLE_OAUTH_ANDROID_CLIENT_ID ?? ''
+      : '',
+  googleRedirectURI:
+    Platform.OS === 'ios'
+      ? `${Config.GOOGLE_OAUTH_IOS_URI_SCHEME}:/oauth2redirect` ?? ''
+      : Platform.OS === 'android'
+      ? `${Config.GOOGLE_OAUTH_ANDROID_CLIENT_ID}:/oauth2redirect` ?? ''
+      : '',
```

Comment:
I personally found them more readable, but maybe it's because of my functional background. What do you suggest as an alternative, if-else? Before the map, that works as well. What do you find more readable? 
https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340

Reply:
Please use explicit if or case statements.

Reply:
I have changed this however I believe it was more readable how it used to be